circle.yml

```diff
@@ -1,5 +1,5 @@
 test:
   override:
-    - npm run jshint
+    - node_modules/jsxhint/cli.js 'app/**/*.js' 'mock/**/*.js' 'test/**/*.js' '*.js'
```

Comment:
IIRC Circle CI was not running jsxhint when calling `npm run jshint` so I had to specify the command explicitly.